x/multisig/abci.go

```diff
@@ -26,14 +26,14 @@ func handleKeygens(ctx sdk.Context, k types.Keeper, rewarder types.Rewarder) {
 	for _, keygen := range k.GetKeygenSessionsByExpiry(ctx, ctx.BlockHeight()) {
 		k.DeleteKeygenSession(ctx, keygen.GetKeyID())
 
+		slices.ForEach(keygen.GetMissingParticipants(), rewarder.GetPool(ctx, types.ModuleName).ClearRewards)
```

Comment:
Can you check the test and update it? I don't like it that this change did not break any test.